static/pages/redirect.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
```

Comment:
It's interesting to see that you chose the worst possible example for creating the html page when you could have used React :D 
(for the record: wexond/renderer/app/index.tsx -> "app" entry point in webpack.renderer.js + HTMLWebpackPlugin)

Reply:
I didn't know it. 😵